client/landing/stepper/declarative-flow/tailored-ecommerce-flow.ts

```diff
@@ -48,6 +48,7 @@ function getPlanFromRecurType( recurType: string ) {
 
 const ecommerceFlow: Flow = {
 	name: ECOMMERCE_FLOW,
+	isSignupFlow: true,
```

Comment:
`isSignupFlow` should be `false` here since the flow will trigger the event on its own rather than depending on Stepper to do it.